samples/cpp/visual_language_chat/visual_language_chat.cpp

```diff
@@ -26,16 +31,18 @@ int main(int argc, char* argv[]) try {
 
     pipe.start_chat();
     std::cout << "question:\n";
+
     std::getline(std::cin, prompt);
-    pipe.generate(
-        prompt,
-        ov::genai::image(image),
-        ov::genai::streamer(print_subword)
-    );
+    auto resuls = pipe.generate(prompt,
+                                ov::genai::images(images),
+                                ov::genai::generation_config(ov::genai::greedy()),
```

Comment:
as I wrote - we will deprecate predefined generation configs. So, let's avoid usage of greedy() and keep just max_new_tokens

Reply:
removed